docs/guides/kernel.md

```diff
@@ -40,8 +40,16 @@ done
 
 ### Setting kernel configuration
 
+There are a few places you could get your kernel config.
+
+- Use the config of the kernel that is currently running (if your running kernel is an older longterm kernel, there could be issues caused by updating its kernel config with the defaults for new options): `zcat /proc/config.gz > .config`
+- Use the Arch Linux kernel config (may have differences to the normal config of your distro, but is up to date): `cp ../patches/config .config`
```

Comment:
Arch Linux kernel config shows issues with Ubuntu. I tried to compile using it once and ended up having just a terminal at boot.

Reply:
I'm not sure how we'd get up to date configs for every distro then, as using one distro's config for another can make it not boot at all.

Reply:
probably stick to `zcat /proc/config.gz > .config` then